### DISCUSSION TITLE
类SQL语法是否限定为ppt给出的基础语法

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#29] [Q&A] [answered]
如题，能否实现一套**兼容**ppt给出语法的，但有有额外可选功能的扩展语法  
带来的不同点包括：某些情况下，原语法要执行两次语句，新语法只用执行一次

**Comment by binbin00** [comment, marked as answer]
想法很好，但是请用ppt里给定的语法，这里的语法也不是我们自己造的，是从SQL截取的。为了控制整个项目的工作量，我们特地只用了一些简单的语法。

**Reply by ghost** [reply]
所以在command.txt中输出 已经被实现的 扩展语法是不被允许的，对吗？
例如ORDER BY不能自己实现并使用，必须先SELECT，然后再排序？

**Reply by binbin00** [reply]
你这属于拓展功能了，你想怎么实现就怎么实现，但是不要对基础功能的要求进行改动。

**Reply by ghost** [reply]
> 你这属于拓展功能了，你想怎么实现就怎么实现，但是不要对基础功能的要求进行改动。

明白了，谢谢
